helix-core/src/main/java/org/apache/helix/tools/ClusterStateVerifier.java

```diff
@@ -181,6 +181,12 @@ private BestPossAndExtViewZkVerifier(HelixZkClient zkClient, String clusterName,
       this.resources = resources;
     }
 
+    public void close() {
```

Comment:
Can we put this into a separate PR? i.e, add close method to ClusterStateVerifier.

Reply:
#1445 is created for this one.